WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -0,0 +1,171 @@
+<% provide(:title, @competition.name) %>
+
+<% if @admin_view %>
+  <h2>Announcements</h2>
```

Comment:
Don't forget about the buttons to post announcements here... or is there a link elsewhere?